README.md

```diff
@@ -13,6 +13,8 @@ This project aims to create an Apache-licensed library for executing eBPF progra
 
 [Instruction set reference](https://github.com/iovisor/bpf-docs/blob/master/eBPF.md)
 
+[API Documentation](http://iovisor.github.io/ubpf/ubpf_8h.html)
```

Comment:
```suggestion
[API Documentation](https://iovisor.github.io/ubpf/ubpf_8h.html)
```

We should definitely use a `https:` link.

Would this be better to go to the top-level URL? https://iovisor.github.io/ubpf/

My concern is that the header page link looks fragile and easily broken if we change the docs.  I don't know how doxygen crates page file names and how stable it is.  Although the docs for doxygen seem to suggest that file names are stable if `SHORT_NAMES` is set to `NO` in the `Doxyfile`.  Which it is.

Happy for you to change or not as you think best.